bundles/org.openhab.binding.yeelight/README.md

```diff
@@ -39,6 +39,10 @@ All devices support some of the following channels:
 |`color` | `Color` | This channel supports color control, it is available on `wonder` and `stripe`.|
 |`colorTemperature` | `Dimmer` | This channel supports adjusting the color temperature, it is available on `wonder` and `stripe` and `ceiling`.|
 |`command` | `String` | This channel sends a command directly to the device, it is available on all Yeelight Things.|
+|`backgroundColor` | `Color` | This channel supports color control for the ambient light, it is available on `ceiling4`.|
+|`backgroundBrightness` | `Dimmer` | This channel supports adjusting the ambient light brightness value, it is available on `ceiling4`.|
```

Comment:
`backgroundBrightness` should probably not be a separate channel. The convention is that a user can create a Dimmer item on the color channel and that should be handled as setting the brightness.

Reply:
I wasn't aware of this convention. I will adapt the code.

Reply:
I can imagine, there is a lot to know. For this see https://www.openhab.org/docs/concepts/items.html#a-note-on-items-which-accept-multiple-state-data-types

Reply:
@Hilbrand I've implemented the changes as suggested and combined ambient light color and brightness into one channel.